### PR TITLE
Document explaining SAM CLI's project operations

### PR DESCRIPTION
**Pull request by sanathkr**
*Issue #, if available:*
The way we operate this project is not transparent to contributors. New contributors do not know where to find help, where to hangout with the team, when releases happen, which new issues can be worked on etc. This document aims to clarify some of these questions and help set the course for how the core team also operates. We should eventually merge this document into `DEVELOPMENT_GUIDE.rst` or `CONTRIBUTING.rst`.

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: OPERATIONS.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+
+Core Team
+---------
+SAM CLI project is still small enough that we don't have a official team structure or governance. Therefore "Core Team" simply consists of developers who have commit access to the this Github Repository. They know most about the source code and work on it every day. 
+
+
+Contributors
+------------
+We are actively looking for contributors to join the project. You could help us fix bugs, write features, improve documentation or even do Marketing/PR activities. Read the DEVELOPMENT_GUIDE.rst file for information on how to get started.
```
Comment (jfuss) [inline OPERATIONS.rst]: Should this be Read the DEVELOPMENT_GUIDE or the CONTRIBUTING or both?

File: OPERATIONS.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+--------------
+Final section of a design document must be breakdown of tasks necessary to implement the design. Once the design document is accepted, create a Github Issue for each task, assign a feature label to the issue, and update the design document with a link to the Issue. This will help any contributor discover and pick up remaining work necessary to complete the feature.
+
+
+Action Items
+============
+
+- [ ] Add a PR Checklist to Pull Request Template (owner?)
+- [ ] Add a PR Reviewer checklist to DEVELOPMENT_GUIDE.rst (owner?)
+- [ ] Create a Design Document teamplate (owner?)
```
Comment (jfuss) [inline OPERATIONS.rst]: Should add an item here to define the time period for meetings: "Every week the core team (a fixed time slot every week) engages in a slack meeting that discusses the following"

**Comment by jfuss** [comment]
@sanathkr and @TheSriram Do either of you know that status of this?

File: OPERATIONS.rst (hunk 1 above)
Comment (sanathkr) [reply OPERATIONS.rst]: Updated

File: OPERATIONS.rst (hunk 2 above)
Comment (sanathkr) [reply OPERATIONS.rst]: Updated

**Comment by jfuss** [comment]
Closing as this is no longer relevant.

**Pull request closed without merging**